lib/collection/response.js

```diff
@@ -167,6 +167,13 @@ var util = require('../util'),
  * @property {String} [body]
  * @property {Buffer|ByteArray} [stream]
  * @property {Number} responseTime
+ * @property {Object} [timings] - timing information related to request and response in milliseconds
+ * @property {Number} [timings.start] - timestamp of the start of the request (in Unix Epoch milliseconds)
+ * @property {Number} [timings.socket] - when the http module's socket event fires (relative to timings.start)
```

Comment:
Instead of referring to the Node's modules the description should describe what `socket`(and others) time represents.